Utils.py

```diff
@@ -240,6 +241,7 @@ def set_movie_properties(json_response):
         window.setProperty('Set.Movie.%d.Path' % count, media_path(item['file']))
         window.setProperty('Set.Movie.%d.Year' % count, str(item['year']))
         window.setProperty('Set.Movie.%d.Duration' % count, str(item['runtime'] / 60))
+        window.setProperty('Set.Movie.%d.Resolution' % count, streaminfo["videoresolution"])
         window.setProperty('Set.Movie.%d.Art(clearlogo)' % count, art.get('clearlogo', ''))
```

Comment:
Done ;)

Thanks for adding this on your code.

2015-02-24 19:08 GMT+01:00 Philipp Temminghoff <notifications@github.com>:

> In Utils.py
> <https://github.com/phil65/script.skin.info.service/pull/3#discussion_r25275725>
> :
>
> > @@ -240,6 +241,7 @@ def set_movie_properties(json_response):
> >          window.setProperty('Set.Movie.%d.Path' % count, media_path(item['file']))
> >          window.setProperty('Set.Movie.%d.Year' % count, str(item['year']))
> >          window.setProperty('Set.Movie.%d.Duration' % count, str(item['runtime'] / 60))
> > +        window.setProperty('Set.Movie.%d.Resolution' % count, streaminfo["videoresolution"])
> >          window.setProperty('Set.Movie.%d.Art(clearlogo)' % count, art.get('clearlogo', ''))
>
> Could you update this PR to use VideoResolution?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phil65/script.skin.info.service/pull/3/files#r25275725>
> .
>